client/galaxy/scripts/components/citations.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+    <div class="toolForm">
+        <div class="toolFormTitle">
+            Citations
+            <button v-if="viewRender" v-on:click="toggleViewRender" type="button" class="btn btn-xs citations-to-bibtex" title="Show all in BibTeX format.">
+                <i class="fa fa-pencil-square-o"></i>
+                Show BibTeX
+            </button>
+            <button v-else type="button" v-on:click="toggleViewRender" class="btn btn-xs citations-to-formatted" title="Return to formatted citation list.">
+                <i class="fa fa-times"></i>
+                Hide BibTeX
+            </button>
+        </div>
+        <div class="toolFormBody" style="padding:5px 10px">
+            <div v-if="source == 'history'" style="padding:5px 10px">
+                <b>Warning: This is a experimental feature.</b> Most Galaxy
+                tools will not annotate citations explicitly at this time. When
+                writing up your analysis, please manually review your histories
+                and find all references that should be cited in order to
+                completely describe your work. Also, please remember to <a
+                href="https://galaxyproject.org/citing-galaxy">cite Galaxy</a>.
+            </div>
+        </div>
+        <div class="citations-bibtex toolFormBody" style="padding:5px 10px">
```

Comment:
Citations.vue template uses same inline style `style="padding:5px 10px" `more than once. Can we use the same css class at these places?
Thanks!

Reply:
Yeah, we shouldn't use inline styles anywhere like this (for the most part).  This was a quick translation from the old backbone version without any updates.

Reply:
(in fact, these styles will be pushed into the `<style>` component block)